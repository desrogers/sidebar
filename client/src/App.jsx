import React from "react";
import ErrorBoundary from "./ErrorBoundary";
import Sidebar from "./components/Sidebar";

export default () => (
  <ErrorBoundary>
    <Sidebar />
  </ErrorBoundary>
);
