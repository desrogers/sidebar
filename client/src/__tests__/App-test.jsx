import React from 'react';
import { shallow, mount, render } from 'enzyme';

import App from '../App';

describe('Hello World', () => {
  it('should render without throwing an error', () => {
    expect(shallow(<App />).contains(<div>Hello World!</div>)).toBe(true);
  });

  it('should render correctly with no props', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
