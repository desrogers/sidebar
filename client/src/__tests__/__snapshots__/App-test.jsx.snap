// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World should render correctly with no props 1`] = `
<div>
  Hello World!
</div>
`;
